python_ta/checkers/type_annotation_checker.py

```diff
@@ -0,0 +1,57 @@
+"""checker for type annotation.
+"""
+
+import astroid
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+
+
+class TypeAnnotationChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker
+
+    name = 'TypeAnnotationChecker'
+    msgs = {'E9970': ('Function argument is missing type annotation',
```

Comment:
Use "parameter" rather than "argument" (and in the short identifier for this error as well).